ford/templates/index.html

```diff
@@ -84,54 +84,54 @@ <h4>{{ author }}</h4>
         <div class="col-xs-6 col-sm-{{ width }}">
               <h3>Source Files</h3>
               <ul>
-                {% for src in project.allfiles|sort(attribute='name') %}
+                {%- for src in (project.allfiles|sort(attribute='name'))[:10] -%}
```

Comment:
Would this cause problems if there are fewer than 10 objects of a particular type? Should this be `min(10,project.allfiles|length)` or similar?

Reply:
I've tested on a project with only a few files etc., and it didn't cause any problems. Turns out Python handles this automatically:

```python
>>> foo = [1, 2]
>>> foo[:10]
[1, 2]
```